lib/system/gc_interface.nim

```diff
@@ -13,6 +13,23 @@ when hasAlloc:
       gcOptimizeTime,    ## optimize for speed
       gcOptimizeSpace    ## optimize for memory footprint
 
+type
+  GCStatsData* = object
+    cycleCollections*: int    # number of performed full collections
+    cycleTableSize*: int      # max entries in cycle table
+    maxCycleTableSize*: int64
+    maxPause*: int64          # max me
+    maxPauseTime*: int64
+    maxStackCells*: int       # max stack cells in ``decStack``
+    maxThreshold*: int        # max threshold that has been set
+    stackCells* : int64
```

Comment:
You should keep the `:` next to the identifier. There are many other places with this style issue in this PR as well.

```suggestion
    stackCells*: int64
```